### PR TITLE
Update newer versions of dependencies

### PR DESCRIPTION
**Pull request by palpalani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,31 +27,31 @@
...
-    "gulp-uglify": "^1.1.0",
-    "imagemin-pngcrush": "^4.0.0",
-    "jshint-stylish": "^1.0.1",
-    "lazypipe": "^0.2.2",
+    "gulp-jshint": "^1.11.0",
+    "gulp-less": "^3.0.3",
+    "gulp-minify-css": "^1.1.5",
+    "gulp-plumber": "^1.0.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-rev": "^4.0.0",
```
Comment (austinpray) [inline package.json]: 4.0.0 doesn't seem to break much https://github.com/sindresorhus/gulp-rev/compare/2f9d798c4d8f4c91bec215df68bfcc3850caa34f...4.0.0

**Comment by retlehs** [comment]
could you please rebase this and squash into one commit?

**Comment by swalkinshaw** [comment]
@palpalani are you able to rebase and squash this?

**Comment by palpalani** [comment]
@swalkinshaw i updated and did rebase and squash

**Pull request merged**